### ISSUE TITLE
[$10][MSFT-71] [Member] Linked image missing alternative text in User Profile Icon

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2785]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Inspect Element
3. Observe the alternative text in User Profile Icon.

**Expected Result**
Linked image alternative text should be present in User Profile Icon

**Actual Result**
Linked image alternative text is missing in User Profile Icon. 

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2071

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096296 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096296 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096296 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu I can't seem to find the component referenced here either in accounts repo nor community-app, can you please verify its present there

**Comment by veshu** [comment]
This is not in scope.
cc @mishacucicea

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097904 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
